apps/api/src/yjs/v2/executor/convertToNotebook.ts

```
@@ -0,0 +1,263 @@
+          metadata = { variables, id: cellId }
+        }
+        notebookCells.push({
+          id: cellId,
+          cell_type: cell_type,
+          source: markdown,
+          metadata: metadata,
+        })
+        break
+      }
+      case 'SQL':
+        const sqlBlock = getSQLAttributes(currentBlock as Y.XmlElement<SQLBlock>, blocks)
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)